### PR TITLE
Quad tilt vtol config

### PR DESCRIPTION
**Pull request by samay**
Added a new vehicle configuration for a Quad Tilt Aircraft

- Vehicle ID
- Main Mixer
- Aux Mixer

### CODE REVIEW

File: ROMFS/px4fmu_common/mixers/claire.aux.mix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Elevator mixers
+
+-------------
+
+M: 1
+
+O:      10000  10000      0 -10000  10000
+
+S: 1 1 -10000 -10000      0 -10000  10000
```
Comment (tumbili) [inline ROMFS/px4fmu_common/mixers/claire.aux.mix]: @samay Please add new line

File: ROMFS/px4fmu_common/mixers/claire.main.mix [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# CruiseAder Claire Main Multirotor mixer for PX4FMU
+
+#
+
+#===========================
+
+R: 4x 10000 10000 10000 0
```
Comment (tumbili) [inline ROMFS/px4fmu_common/mixers/claire.main.mix]: Please add new line

**Comment by tumbili** [comment]
@samay Thanks for the PR! If you add the missing new lines and rebase on master then I'll get this in.

**Comment by samay** [comment]
@tumbili thank you , i added the new lines but not sure why the sitl build failed ?

**Pull request merged**